### PR TITLE
WFP-1549: New namespace for staff information

### PR DESCRIPTION
**Pull request by sp-watson**

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-staff-dev/01-rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: hmpps-staff-dev-admin
+  namespace: hmpps-staff-dev
+subjects:
+  - kind: Group
```
Comment (carlov20) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-staff-dev/01-rbac.yaml:7]: this also needs dps-tech added too

```
  - kind: Group
    name: "github:dps-tech"
    apiGroup: rbac.authorization.k8s.io
```

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-staff-dev/05-certificate.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hmpps-staff-dev.hmpps.service.justice.gov.uk
+  namespace: hmpps-staff-dev
+spec:
+  secretName: hmpps-staff-cert
```
Comment (carlov20) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-staff-dev/05-certificate.yaml:8]:
```suggestion
  secretName: hmpps-staff-api-cert
```

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-staff-dev/05-certificate.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+    - hmpps-staff-dev.hmpps.service.justice.gov.uk
```
Comment (carlov20) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-staff-dev/05-certificate.yaml:13]:
```suggestion
    - hmpps-staff-api-dev.hmpps.service.justice.gov.uk
```

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-staff-dev/05-certificate.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hmpps-staff-dev.hmpps.service.justice.gov.uk
```
Comment (carlov20) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-staff-dev/05-certificate.yaml:5]:
```suggestion
  name: hmpps-staff-api-dev.hmpps.service.justice.gov.uk
```

**Review by vijay-veeranki — APPROVED** [review]

**Pull request merged**